query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/GraphQLTransformOutput.java

```diff
@@ -167,7 +170,7 @@ public Object functionDataFetcher(DataFetchingEnvironment env, FunctionField<Dat
         FunctionReport<DataType> result = funcField.getValue();
 
         if(!result.messages().isEmpty()) {
-            throw new FunctionDataFetcherException(funcField.fieldName(), args, result.messages());
+            throw new FunctionDataFetcherException(toPrettyString(funcField, result.messages()), result.messages());
```

Comment:
The thing with moving it into `FunctionDataFetcherException` is you can't call super with it and that is what we want to do.